cli/install/viz.go

```diff
@@ -0,0 +1,67 @@
+package install
+
+// Viz defines the primary Conduit Grafana dashboard, installed via the `conduit install` command.
+const Viz = `{
+      "rows": [
+        {
+          "collapse": false,
+          "height": "50px",
+          "panels": [
+            {
+              "content": "<div>\n  <div style=\"position: absolute; top: 0, left: 0\">\n    <a href=\"https://conduit.io\" target=\"_blank\"><img src=\"https://conduit.io/images/conduit-primary-white.svg\" style=\"height: 30px;\"></a>\n  </div>\n  <div style=\"position: absolute; top: 0; right: 0; font-size: 15px\">\n    <a href=\"https://conduit.io\" target=\"_blank\">Conduit</a> is a next-generation ultralight service mesh for Kubernetes.\n    <br>\n    Need help? Join us on <a href=\"http://slack.linkerd.io\" target=\"_blank\">Slack</a>.\n  </div>\n</div>",
```

Comment:
TIOLI, rather than linking this message to slack.linkerd.io, maybe we could just link it to conduit.io instead? Something like:

> Need help? Visit [conduit.io](https://conduit.io).

Reply:
![screen shot 2018-02-28 at 11 15 30 am](https://user-images.githubusercontent.com/236915/36807875-d45b50a6-1c78-11e8-9607-a93f377a4538.png)